pkg/customresource/repositoryserver.yaml

```diff
@@ -44,12 +44,9 @@ spec:
                       to the kopia repository
                     properties:
                       content:
-                        type: string
+                        type: integer
```

Comment:
Should we add description, about what unit would be used for the provided value?

Reply:
makes sense, I have added it